resteasy-client/src/main/java/org/jboss/resteasy/plugins/providers/sse/client/SseEventSourceImpl.java

```diff
@@ -208,12 +199,6 @@ public boolean isOpen()
       return state.get() == State.OPEN;
    }
 
-   @Override
```

Comment:
If this method is removed, default jax-rs close method is used, right? Default method from spec uses 5s timeout:
```java
    @Override
    default void close() {
        close(5, TimeUnit.SECONDS);
    }
```

Reply:
Yes, it's right